src/wp-content/themes/twentytwentytwo/inc/block-patterns.php

```diff
@@ -3,6 +3,7 @@
  * Twenty Twenty-Two: Block Patterns
  *
  * @since Twenty Twenty-Two 1.0
+ *
```

Comment:
```suggestion
```

The file header does not need another line under `@since`. 
https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#6-file-headers